modules/new_profile/php/new_profile.class.inc

```diff
@@ -43,62 +43,42 @@ class New_Profile extends \NDB_Form
         return false;
     }
     /**
-     * Processes the values and saves to database
+     * Processes form data and creates a new candidate.
      *
      * @param array $values form values
      *
      * @return void
      */
-    function _process($values)
```

Comment:
This function was causing a lot of bugs with regard to typing so I rewrote it to use strict types and a clearer style of operating.